src/components/LandingPage/LandingPage.tsx

```diff
@@ -1,14 +1,16 @@
 "use client";
 
 import env from "@/env";
-import LoginButton from "../LoginButton";
+import ConnectButton from "../ConnectButton/ConnectButton";
```

Comment:
please add `index.tsx` file to the `/ConnectButton/` directory so we do not have this double-nested component name
- actual: `./ConnectButton/ConnectButton`
- expected: `./ConnectButton`